src/client/Microsoft.Identity.Client/Internal/Logger/DurationLogHelper.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using Microsoft.Identity.Client.Core;
+
+namespace Microsoft.Identity.Client.Internal.Logger
+{
+    internal sealed class DurationLogHelper : IDisposable
```

Comment:
@neha-bhargava - the IDisposable pattern is very common in C# (somewhat similar to AutoCloseable in Java)... because it plays very nicely with the `using` statement - so code like is quite common.

`using (var dh = new DurationHelper())` is syntactic sugar for 

```csharp
try
{
var dh = new DurationHelper();
}
finally
{
  dh.Dispose();
}
```